### PR TITLE
API for deleting a user from a Participant Team.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#336 solved. @deshraj Please review the PR.

### CODE REVIEW

File: apps/participants/urls.py [hunk 1]
```diff
@@ -14,4 +14,7 @@
     url(r'participant_teams/challenges/user',
         views.get_teams_and_corresponding_challenges_for_a_participant,
         name='get_teams_and_corresponding_challenges_for_a_participant'),
+    url(r'(?P<participant_team_pk>[0-9]+)$',
```
Comment (deshraj) [inline apps/participants/urls.py]: Can you prepend `participant_teams/` in the URL?

Comment (RishabhJain2018) [inline apps/participants/urls.py]: yeah sure.

**Comment by TroJan** [comment]
@RishabhJain2018 : I have reviewed Pr #343 . the changes suggested over there applies here also, just that instead of challenge host and challenge host team, concerned entities will be participant and participant team. I can clarify your doubts in case you face to get this pr in sync with challenge host team one. Please feel free to ask or comment.

File: tests/unit/participants/test_views.py [hunk 2]
```diff
@@ -463,3 +463,38 @@ def test_when_there_is_no_participant_team_of_user(self):
         response = self.client.get(self.url, {})
         self.assertEqual(response.data, expected)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+
+class RemoveParticipantFromTeamHimselfTest(BaseAPITestClass):
```
Comment (deshraj) [inline tests/unit/participants/test_views.py]: Please rename this to `RemoveSelfFromParticipantTeam`

**Pull request merged**